packages/modular-scripts/src/__tests__/index.test.ts

```diff
@@ -204,11 +204,11 @@ describe('modular-scripts', () => {
             "README.md",
           ],
           "license": "UNLICENSED",
-          "main": "dist-cjs/sample-view.cjs.js",
+          "main": "dist-cjs/index.js",
           "modular": Object {
             "type": "view",
           },
-          "module": "dist-es/sample-view.es.js",
+          "module": "dist-es/index.js",
           "name": "sample-view",
           "typings": "dist-types/src/index.d.ts",
```

Comment:
https://github.com/jpmorganchase/modular/pull/608/files#diff-4870b5ce0bd4c0a47c7dfc47c42422d6e2455d523069b2f0cfbeaf1a70e759b9R33 removed the src folder so you just need update the typings path

Reply:
Resolved this and added a test case for it!